metadata.json

```diff
@@ -2,39 +2,60 @@
   "name": "thias-sysctl",
   "version": "1.0.7",
   "author": "Matthias Saou",
-  "license": "Apache-2.0",
   "summary": "Sysctl module",
+  "license": "Apache-2.0",
   "source": "git://github.com/thias/puppet-sysctl.git",
   "project_page": "https://github.com/thias/puppet-sysctl",
   "issues_url": "https://github.com/thias/puppet-sysctl/issues",
-  "tags": [ "sysctl", "os" ],
+  "dependencies": [
+
+  ],
   "operatingsystem_support": [
     {
-    "operatingsystem": "RedHat",
-    "operatingsystemrelease": [ "5", "6", "7", "8" ]
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
+        "5",
+        "6",
+        "7",
+        "8"
+      ]
     },
     {
-    "operatingsystem": "CentOS",
-    "operatingsystemrelease": [ "5", "6", "7", "8" ]
+      "operatingsystem": "CentOS",
+      "operatingsystemrelease": [
+        "5",
+        "6",
+        "7",
+        "8"
+      ]
     },
     {
-    "operatingsystem": "Debian",
-    "operatingsystemrelease": [ "6", "7", "8" ]
+      "operatingsystem": "Debian",
+      "operatingsystemrelease": [
+        "6",
+        "7",
+        "8"
+      ]
     },
     {
-    "operatingsystem": "Ubuntu",
-    "operatingsystemrelease": [ "12", "14" ]
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "12",
+        "14"
+      ]
     }
   ],
   "requirements": [
     {
-    "name": "pe",
-    "version_requirement": "3.x"
-    },
-    {
-    "name": "puppet",
-    "version_requirement": ">=2.7.20 <8.0.0"
+      "name": "puppet",
+      "version_requirement": ">=2.7.20 < 9.0.0"
```

Comment:
Minor thing but version_requirement is probably at least >= 4.0

Reply:
I guess the module will still be compatible with historic versions of Puppet. I guess therefore the maintainer didn't include stdlib.

Reply:
I recommend to first merge this PR into our fork and create a new PR afterwards for the compatibility breaking changes.